src/index.ts

```diff
@@ -43,6 +44,7 @@ export {
   FileUploadButton,
   Spinner,
   Tab,
+  Countdown,
```

Comment:
Let's export `useCountdownTimer` hook as well in case the requirements are so different that they cannot use `Countdown` component